### PR TITLE
LTD-749 update outdated jquery

### PR DESCRIPTION
**Pull request by r4vi**
This turned into quite the mammoth task so I have made the commits as atomic as possible.

The problem this PR tried to solve was the usage of an outdated vendored version of Jquery but this was a can of worms.

To use the jquery bundled via node_modules, a lot of the dependent modules needed to be updated to be able to interact with a version of jquery that is provided by a JS module system rather than just being a global available as `$`

I've also updated our javascript bundler `parcel` to the latest version

### CODE REVIEW

File: caseworker/templates/components/picklist-picker.html [hunk 1]
```diff
@@ -42,8 +42,8 @@
 
 	</div>
 {% endif %}
-
-<script>
+{% block javascript %}
+<script nonce="{{ request.csp_nonce }}">
```
Comment (agalamatis) [inline caseworker/templates/components/picklist-picker.html:46]: No script type?

**Comment by agalamatis** [comment]
I think there is [one more place](https://github.com/uktrade/lite-frontend/blob/master/exporter/templates/applications/goods-types/countries.html#L76) helpers.js is pulled directly.

**Review by agalamatis — APPROVED** [review]
I left some comments about a couple of minor things.

On the assumption that I trust node generating everything correctly, it addresses the issue identified by the PenTest regarding jQ.

File: caseworker/templates/components/picklist-picker.html (hunk 1 above)
Comment (r4vi) [reply caseworker/templates/components/picklist-picker.html:46]: just staying consistent with the rest of the code base.

**Comment by r4vi** [comment]
> I think there is [one more place](https://github.com/uktrade/lite-frontend/blob/master/exporter/templates/applications/goods-types/countries.html#L76) helpers.js is pulled directly.

looks like that's on master, removed in this branch

**Pull request merged**